lib/kakine/config.rb

```diff
@@ -0,0 +1,39 @@
+require 'yao'
+require 'yaml'
+
+module Kakine
+  class Config
+    def self.setup
```

Comment:
`Kakine::Config.setup` must be explicitly called when client is invoked!

May be @ `lib/kakine/cli.rb`